ui/src/app/job-list/job-list.component.spec.ts

```diff
@@ -44,6 +45,8 @@ describe('JobListComponent', () => {
 
   // Jobs with IDs JOB0 -> JOB4.
   function testJobs(count: number): QueryJobsResult[] {
+    const flags = ['archive','','archive','','archive'];
```

Comment:
opt: `new Array(3).fill('archive');`

Reply:
actually, this isn't needed anymore, so I removed it altogether